pyvsystems/chain.py

```diff
@@ -53,13 +51,13 @@ def check_with_other_node(self, node_host, super_node_num=DEFAULT_SUPER_NODE_NUM
         try:
             h1 = self.height()
         except NetworkException:
-            self.logger.error("Fail to connect {}.".format(node_host))
+            # "Fail to connect {}".format(node_host)
```

Comment:
`.format(node_host)` is meanness. Change comment to `Fail to connect full node.`